cmd/sriovdp/manager.go

```diff
@@ -208,28 +210,57 @@ func (rm *resourceManager) discoverHostDevices() error {
 			}
 			glog.Infof("discoverDevices(): device found: %-12s\t%-12s\t%-20s\t%-40s", device.Address, device.Class.ID, vendorName, productName)
 
-			// exclude device in-use in host
-			if isDefaultRoute, _ := hasDefaultRoute(device.Address); !isDefaultRoute {
+			// Network devices
+			if devClass == netClass {
```

Comment:
consider refactoring the two branches into their own handler functions for increased readability